scripts/New-VersionConstantsFile.ps1

```diff
@@ -24,7 +30,7 @@ namespace $namespace
     {
         public const string Prerelease = "$versionSuffix";
         public const string AssemblyVersion = "$versionPrefix";
-        public const string FileVersion = AssemblyVersion + ".0";
+        public const string FileVersion = AssemblyVersion + Prerelease + "$packageVersionSuffix";
```

Comment:
>FileVersion [](start = 28, length = 11)

This might work but it isn't "right". `FileVersion` was intended to be the "dotted quad" .NET assembly version number _e.g._ `2.0.0.0`.

Reply:
Nuts, yes. i wonder if this would break reflecting over our metadata will go take a look.

---
In reply to: [259151626](https://github.com/Microsoft/sarif-sdk/pull/1304#discussion_r259151626) [](ancestors = 259151626)